charts/icm-as/values.yaml

```diff
@@ -560,6 +559,11 @@ jgroups:
   # dns_ping (playground) declared dns query
   discovery: file_ping
 
+  # location allows to set a different (optional) directory for file_ping discovery protocol
```

Comment:
Introducing a new configuration requires a minor release according to SemVer for backward-compatibility, please adjust labels and PR title (later for merge commit) to use `feat(icm): [...]`

Reply:
@smoldenhauer-ish The prefix `feat(icm-as)` with the hyphen in `icm-as` would not be detected by the current regex configuration for semantic versioning action: https://github.com/intershop/helm-charts/blob/main/.github/workflows/release-branch.yml#L70

Reply:
Ok, changed. 
And I hope the labels, title and commit are now as it should be.

Reply:
Ok, got it now. Maybe. Confusing it with the labels.